### PR TITLE
{bio}[foss/2016a,intel/2016a] PHYLIP v3.696

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2715.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/204f8747e3e89ec224c9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/93d35631a6b11860c9b7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2715.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ae73dc6f2fc1b7844c60 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/134164cbef6db2158b37 for a full test report.

File: easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.696-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name = 'PHYLIP'
+version = '3.696'
+
+homepage = 'http://evolution.genetics.washington.edu/phylip'
+description = "PHYLIP is a free package of programs for inferring phylogenies."
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://evolution.gs.washington.edu/phylip/download/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/PHYLIP/PHYLIP-3.696-foss-2016a.eb]: `SOURCELOWER_TAR_GZ`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2074.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f852d5377516f80cfee2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2715.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4309f63b375f3f23d6b2 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**